### DISCUSSION TITLE
Towny plugin's language folder -> reference folder is empty.

### DISCUSSION THREAD

**Discussion by junyeonglee00** [TownyAdvanced/Towny#7527] [Q&A] [answered]
![화면 캡처 2024-07-26 184115](https://github.com/user-attachments/assets/5c40b47c-2970-4100-ac4d-83b2637807f1)

**Comment by LlmDl** [comment, marked as answer]
Towny only writes the reference files when the Towny version is changed. If you want the files written open your config.yml and set the last run version to a version lower than the one you're on. When Towny sees that you're updating versions, it will write the reference files.
